CONTRIBUTING.md

```diff
@@ -331,93 +330,6 @@ satisfy the tests provided for each problem.
 You can include advice and helpful links for your language track in the
 `SETUP.md` file.
 
-## Maintaining a Track
```

Comment:
Is it better to add a partial review that says 'some information missing' or nothing at all because I don't have time to do a thorough pass?